Tasks/GitHubReleaseV0/task.json

```diff
@@ -261,6 +269,7 @@
         "TagRequiredCreateAction": "Tag source is set to manual- please specify a tag for create action. For yaml syntax see: https://aka.ms/AA3m1bq",
         "NoFileFoundMatchingPattern": "No files found matching '%s'. Nothing to upload.",
         "PatternIsADirectory": "'%s' cannot be uploaded as it is a directory. Please specify a file.",
-        "SearchingFileMatchingPattern": "Searching for file(s) matching '%s'."
+        "SearchingFileMatchingPattern": "Searching for file(s) matching '%s'.",
+        "NoMatchingTagsFound": "No Matching tags found for given regex: %s  Unable to perform the action."
```

Comment:
do these strings need to be reviewed by PM? "Unable to perform the action" doesn't really sound good.

Reply:
Raiyan's OOF currently. Will take this in the next PR along with L0 tests